RoadMap_Movie/RoadMap_Movie/Controllers/MainScreenViewController.swift

```diff
@@ -0,0 +1,201 @@
+    // MARK: - Constans
+
+    private enum Constants {
+        enum MovieSections {
+            static let topRated = "Высокий рейтинг"
+            static let popular = "Популярное"
+            static let actual = "Скоро"
+        }
+
+        enum Colors {
+            static let red = "redMark"
+            static let orange = "orangeMark"
+            static let green = "greenMark"
+            static let blue = "blueButton"
+            static let gray = "grayForUI"
+            static let lightBlue = "lightBlue"
+        }
+
+        static let movieCell = "movieCell"
+        static let error = "error"
+        static let screenTitle = "Movie"
+        static let detailScreenTitle = "Details"
+        static let ruLanguageImage = "ru"
+        static let enLanguageImage = "en"
+    }
+
+    // MARK: - Private visual elements
+
+    private lazy var selectTopRatedMoviesListButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(Constants.MovieSections.topRated, for: .normal)
+        button.tag = 0
+        button.titleLabel?.font = .systemFont(ofSize: 12, weight: .semibold)
+        button.backgroundColor = UIColor(named: Constants.Colors.blue)
+        button.addTarget(self, action: #selector(changeMoviesListAction), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var selectPopularMoviesListButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(Constants.MovieSections.popular, for: .normal)
+        button.tag = 1
+        button.titleLabel?.font = .systemFont(ofSize: 12, weight: .semibold)
+        button.backgroundColor = UIColor(named: Constants.Colors.blue)
+        button.addTarget(self, action: #selector(changeMoviesListAction), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var selectLatestMoviesListButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(Constants.MovieSections.actual, for: .normal)
+        button.tag = 2
+        button.titleLabel?.font = .systemFont(ofSize: 12, weight: .semibold)
+        button.backgroundColor = UIColor(named: Constants.Colors.blue)
+        button.addTarget(self, action: #selector(changeMoviesListAction), for: .touchUpInside)
+        return button
+    }()
+
+    private let tableView = UITableView()
+
+    // MARK: - Private properties
+
+    private let sessionConfiguration = URLSessionConfiguration.default
+    private let decoder = JSONDecoder()
+    private let session = URLSession.shared
+    private var movies: Results?
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private methods
+
+    @objc private func changeMoviesListAction(_ sender: UIButton) {
+        switch sender.tag {
+        case 0:
+            obtainMovies(
+                sectionUrl: UrlRequest.baseURL + UrlRequest.topRated + UrlRequest.apiKey + UrlRequest
```

Comment:
```suggestion
                sectionUrl: UrlRequest.baseURL + UrlRequest.topRated + UrlRequest.apiKey + UrlRequest
```
 убираем конкатенацию
 для таких вещей есть URLComponents